src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/InitializeDampedHarmonic.cpp

```diff
@@ -147,16 +229,48 @@ void InitializeDampedHarmonic<Dim>::DampedHarmonicRollonCompute<Frame>::
       rollon_start_time, rollon_width, sigma_r);
 }
 
+template <size_t Dim, bool UseRollon>
+template <typename Frame>
+void InitializeDampedHarmonic<Dim, UseRollon>::DampedHarmonicCompute<Frame>::
+    function(
+        const gsl::not_null<return_type*> h_and_d4_h,
+        const Scalar<DataVector>& lapse,
+        const tnsr::I<DataVector, Dim, Frame>& shift,
+        const tnsr::a<DataVector, Dim, Frame>& spacetime_unit_normal_one_form,
+        const Scalar<DataVector>& sqrt_det_spatial_metric,
+        const tnsr::II<DataVector, Dim, Frame>& inverse_spatial_metric,
+        const tnsr::aa<DataVector, Dim, Frame>& spacetime_metric,
+        const tnsr::aa<DataVector, Dim, Frame>& pi,
+        const tnsr::iaa<DataVector, Dim, Frame>& phi,
+        const tnsr::I<DataVector, Dim, Frame>& coords,
+        const double sigma_r) noexcept {
+  if (UNLIKELY(h_and_d4_h->number_of_grid_points() != get(lapse).size())) {
+    h_and_d4_h->initialize(get(lapse).size());
+  }
+  damped_harmonic(
+      make_not_null(&get<Tags::GaugeH<Dim, Frame>>(*h_and_d4_h)),
+      make_not_null(&get<Tags::SpacetimeDerivGaugeH<Dim, Frame>>(*h_and_d4_h)),
+      lapse, shift, spacetime_unit_normal_one_form, sqrt_det_spatial_metric,
+      inverse_spatial_metric, spacetime_metric, pi, phi, coords, 1., 1.,
+      1.,       // amp_coef_{L1, L2, S}
```

Comment:
should/are these hard-coded values be documented somewhere?

Reply:
No they are not. I'm going to make them input file options in my next PR. They've been hard-coded for forever and I think it's time to change that :)

Reply:
ok, as long as they are being changed in the next PR...